server/src/routes/index.ts

```diff
@@ -30,7 +31,7 @@ function register(app: express.Application, prefix: string, authenticator: Authe
     res.send("Hello ingress!");
   });
 
-  registerApiRoutes(app, prefix, authenticator);
+  registerApiRoutes(app, prefix + config.api.prefix, authenticator);
```

Comment:
I think it makes sense to add `api` to the prefix before calling `registerApiRoutes`, but is there an advantage to making the path for the API routes a configuration? Will be also make this configurable in the UI client?

Reply:
I thought someone may prefer a different URL structure, e.g. using something different than `/ui-server`. With the `config.api.prefix` config, it's even possible to route all the APIs to something like `api.customrenku.io` for the backend instead of `customrenku.io/<something>`. Not sure if it's very useful.

I kept the distinction between the server prefix and the API prefix because I initially thought the internal APIs may go under a different prefix (e.g. the /version may go under /ui-server/version instead of /ui-server/api/version). On a second thought, the client should not know about this implementation detail.

Reply:
I think allowing the API to be deployed under a different domain could be a possibility, though, I'm not sure how nice it would be in practice, since it would require setting up a bunch of CORS stuff. 

But I don't think the paths to the endpoints should be configurable: the should be fixed and documented, not subject to change between deployments.

So what do you think about making this line and removing the API prefix configuration?

```suggestion
  registerApiRoutes(app, prefix + "/api", authenticator);
```

Reply:
You are right, configurable paths don't make much sense in this case.
As briefly discussed offline, it may still be useful to keep paths in the config file for a few reasons (reusability, filtering in middlewares, ...).
I reorganized them in a coherent way -- all the routing prefixes are now together (we have only 2, api and auth, but we will have more)